README.md

```diff
@@ -734,6 +735,24 @@ The best practice to determine top candidates for changing materialization from
 - if a view is used downstream my *many* models, change materialization to table
 - if a view has more complex calculations (window functions, joins between *many* tables, etc.), change materialization to table
 
+### Exposure Materializations
+#### Model
+
+`fct_exposure_parentes_materializations` ([source](models/marts/performance/fct_exposure_parentes_materializations.sql)) shows each model with a direct relationship to an exposure that is *not* materialized as a table in the warehouse. 
```

Comment:
```suggestion
`fct_exposure_parents_materializations` ([source](models/marts/performance/fct_exposure_parents_materializations.sql)) shows each model that is a direct parent of an exposure and is *not* materialized as a table in the warehouse. 
```

Reply:
i actually wanted this one to be in spanish